src/day02/ArrayShift.java

```diff
@@ -0,0 +1,34 @@
+package day02;
+
+import java.util.Arrays;
+
+public class ArrayShift {
+    public static void main(String[] args) {
+        int[] firstArr = {2, 4, 6, 8};
+        int newVal = 5;
+        insertShiftArray(firstArr, newVal);
+    }
+
+    public static void insertShiftArray(int[] firstArr, int newVal) {
+        int length = firstArr.length;
+//        int[] secondArr = new int[length + 1];
+
+        int[] tempA = new int[(length + 1) / 2];
+        int[] tempB = new int[length - tempA.length];
+
+        for (int i = 0; i < length; i++) {
+            if (i < tempA.length) {
+                tempA[i] = firstArr[i];
+            } else {
+                tempB[i - tempA.length] = firstArr [i];
+            }
+        }
+
+        int[] secondArr = new int[tempA + tempB];
```

Comment:
Since `tempA` and `tempB` are both arrays, you can't simply add or concatenate them together.  There is, however, a way!
```
int[] array1 = new int[]{1,2,3};
int[] array2 = new int[]{4,5,6};
int[] resultArr = ArrayUtils.addAll(array1, array2);

// *resultArr will be [1,2,3,4,5,6];*
```